### PR TITLE
Remove bike lane stuff, move to DOT cameras

### PR DESCRIPTION
**Pull request by Bellspringsteen**
Moving away from bike lane/bus lane to general DOT camera work. 

Feedback on analyzeimage and saveimage and the associated test files would be nice.

### CODE REVIEW

File: saveimages.py [hunk 1]
```diff
@@ -1,41 +1,222 @@
...
+    SaveImages().makeSureDirectoriesExist();
+    pool = Pool(processes=20)              # start 4 worker processes
+    cameraObjects = SaveImages().fillCameraObjectsWithCameraId(SaveImages().getCameraObjectsWithoutCameraId())
+    SaveImages().saveObjectsToFile("/tmp/objects.json",cameraObjects)
+    SaveImages().download_dot_files(pool,cameraObjects)
+    while (True):
+        if SaveImages().returnTrueToDownloadMoreImages(MAX_FILES_TO_DOWNLOAD):
+            SaveImages().download_dot_files(pool,cameraObjects)
+        else:
+            time.sleep(1.0)
```
Comment (patsplat) [inline saveimages.py]: always fret about large main methods.

is there anything that needs to be cleaned up when breaking out of `while(True):`?

File: saveimages.py [hunk 2]
```diff
@@ -1,41 +1,222 @@
...
+DOT_CAMERA_LIST_URL = "https://webcams.nyctmc.org/new-data.php?query="
+DOT_CAMERA_ID_URL = "https://webcams.nyctmc.org/google_popup.php?cid="
+saveDirectory = "/tmp/rawimages/"
+outDirectory = "/tmp/preprocessed/"
+BUCKET = "intersection-ourcamera"
+MAX_FILES_TO_DOWNLOAD = 2000
+NUMBER_FILES_DOWNLOAD_LIMIT = 1000
+save_to_aws = True
+ACCESS_KEY = ""
+SECRET_KEY = ""
```
Comment (patsplat) [inline saveimages.py]: odd mix of args -- save to aws, but no credentials?

File: saveimages.py [hunk 3]
```diff
@@ -1,41 +1,222 @@
...
-* DOT does seem to turn the cameras occasionally i.e. move it to cover a different street
+This class first pings a DOT backend to get the list of location ids, then associates location ids to camera ids, and
+then downloads images.
 
 
 Example usage:
-    ./download_dot_files \
-        --url=http://207.251.86.238/cctv476.jpg?math=0.29640904121642864 \
-        --save_directory=path/to/data_dir
+    python saveimages.py
```
Comment (patsplat) [inline saveimages.py]: how could I download a smidge of the working set?

File: README.md [hunk 4]
```diff
@@ -1,94 +0,0 @@
-# OurCamera
```
Comment (julieyang52) [inline README.md]: We don't have no README.md at all now?

**Pull request merged**